add-domain

```diff
@@ -999,3 +999,32 @@ wiflix.xyz
 wiflix.yachts
 coronavirus.zone
 fullcopec.zone
+dexsceerner.net
+app.dexscreener-home.net
+dexscreener-home.net
+traolerjoe.org
+traderjoexyz.bylfg.org
+bylfg.org
+v2-o-p-e-n-s-e-a.com
+susni-swap.com
+sushi.swap-ether.site
+swap-ether.site
+open-sea.market-ntf.com
```

Comment:
Double of `market-ntf.com`

- https://kb.mypdns.org/issue/MTX-127026/market-ntf.com MTX-127026
- https://mypdns.youtrack.cloud/issue/MTXB-209544/market-ntf.com MTXB-209544
- https://github.com/mypdns/matrix/issues/127026